server/pyproject.toml

```diff
@@ -24,8 +24,8 @@ description = "A search interface for cancer variant interpretations assembled b
 license = {file = "LICENSE"}
 dependencies = [
     "ga4gh.vrs==2.0.0a14",
-    "ga4gh.cat_vrs~=0.3.0",
-    "ga4gh.va_spec~=0.2.0-a1",
+    "ga4gh.cat_vrs~=0.5.0",
+    "ga4gh.va_spec==0.2.0",
```

Comment:
should I revert these since @korikuzma is updating them in the next few days?

Reply:
That's fine, also don't worry about failing tests. I'll be fixing these in my branch